.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+<!--
+Please, don't forget to use the prefixes in your title, following document https://github.com/gofiber/fiber/blob/master/.github/contribuiting.md.
+
+
+e.g
+🔥Feat: Add flow to add person (title)
+
+Added to flow to add a person. (description your pull request)
```

Comment:
We need a better example for this, something like:

```
📚 docs: Improve Middleware docs

Add an example on how to use the embed middleware
```

Also I prefer the old structure of the PR template